src/util/ImageUtil.js

```diff
@@ -1,12 +1,11 @@
 L.ImageUtil = {
 
-  getCmPerPixel(overlay) {
-    var map = overlay._map;
-
-    var dist = map
+  getCmPerPixel:((overlay) =>{
+    let map = overlay._map;
+    let dist = map
         .latLngToLayerPoint(overlay.getCorner(0))
         .distanceTo(map.latLngToLayerPoint(overlay.getCorner(1)));
 
     return (dist * 100) / overlay.getElement().width;
-  },
+  }),
```

Comment:
```suggestion
  },
```

Reply:
@kabuur Please revert these changes and leave only the `var` keyword change. Thanks!